penguin/project/workflow_orchestrator.py

```
@@ -148,9 +144,26 @@ async def run_next_task(self, project_id: Optional[str] = None) -> Optional[Dict
             self._debug("Executing task via ProjectTaskExecutor")
             exec_result = await self.task_executor.execute_task(task)
             self._debug(f"Execution result: {exec_result}")
-            if exec_result["status"] == "error":
+            executor_status = exec_result.get("status")
+            if executor_status == "error":
                 raise Exception(f"Task execution failed: {exec_result['message']}")
```

Ruff (0.15.10)

[warning] Abstract `raise` to an inner function

(TRY301)